src/pages/design/color.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Color
+description: >
+  The Zendesk Garden color system is a set of purposeful colors designed with
+  brand personality, usability, and accessibility in mind. The system is
+  broken down into two core palettes: UI and brand.
+---
+
+## UI colors
+
+The UI colors are used when creating interface elements. Each color has been
+designed to meet accessibility requirements.
+
+### Primary colors
+
+Primary colors are used for the structure of interfaces, actionable items,
+and validation. Colors in the 600, 700, and 800 ranges have a [WCAG
+AA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
+ratio above 4.5:1 against white or 100 backgrounds. The 800 color range
+(excluding yellow) has AAA contrast ratios above 7:1 against white and 100
+backgrounds.
+
+<ColorPalette hues={['grey', 'kale', 'blue', 'green', 'red', 'yellow']} />
+
+### Secondary colors
+
+Secondary colors are used in supplementary UI elements such as icons, tags,
+status badges, and illustrations. Each color has a light and dark hue with a
+default and muted variant. Muted colors are denoted by an "M" prefix, like
+M400.
+
+The default color variants should be used for small UI elements when the
+design requires a vibrant color. The muted variants are for applications with
+larger floods of color like data visualization and infographics. Colors in
+the 400 range meet a minimum contrast of 3:1 on white backgrounds. Colors in
+the 600 range meet a 4.5:1 contrast ratio on white backgrounds.
+
+<ColorPalette
+  hues={[
+    'fuschia',
+    'pink',
+    'crimson',
+    'orange',
+    'lemon',
+    'lime',
+    'mint',
+    'teal',
+    'azure',
+    'royal',
+    'purple'
+  ]}
+/>
+
+## Brand colors
+
+The brand color palette contains the primary brand colors of Zendesk’s
+product suite. These colors are reserved to denote elements in the UI related
+to Zendesk products.
+
+<ColorPalette hues={['product']} />
```

Comment:
Since these colors are reserved for product usage, I wonder if it might be better to visualize them differently from the others. Perhaps showing them as part of a logo display?

Not sure, what do you think @rossmoody? It's also fine to leave as is.

Reply:
@allisonacs are you talking about something like we did in the olden days with https://zendeskgarden.github.io/css-components/utilities/color/?

Reply:
I'm a vote for leaving as-is for now but exploring a different way to visualize the palette in that section might be nice as a follow-on.

Reply:
@jzempel yea, I think that works a bit better. But as @rossmoody says, maybe we do it as a follow-on in our next epic. Your call!